src/app/hearing-part/components/assign-hearing-dialog/assign-hearing-dialog.component.ts

```diff
@@ -27,12 +27,14 @@ export class AssignHearingDialogComponent implements OnInit {
   notes: NoteViewmodel[];
 
   private selectedHearingId: string;
+  startTimeDisplayed: boolean;
 
   constructor(public dialogRef: MatDialogRef<any>,
-              @Inject(MAT_DIALOG_DATA) public data: string,
+              @Inject(MAT_DIALOG_DATA) public data: any,
```

Comment:
instead of any you can create some interface or even use inline type declaration if its only 2 properties